csharp/Apache.Arrow.Flight.Sql.TestWeb/TestFlightSqlServer.cs

```diff
@@ -0,0 +1,182 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Apache.Arrow.Flight.Server;
+using Apache.Arrow.Types;
+using Arrow.Flight.Protocol.Sql;
+using Google.Protobuf;
+using Google.Protobuf.WellKnownTypes;
+using Grpc.Core;
+
+namespace Apache.Arrow.Flight.Sql.TestWeb;
+
+public class TestFlightSqlServer : FlightServer
+{
+    private readonly FlightSqlStore _flightStore;
+
+    public TestFlightSqlServer(FlightSqlStore flightStore)
+    {
+        _flightStore = flightStore;
+    }
+
+    public override async Task DoAction(FlightAction request, IAsyncStreamWriter<FlightResult> responseStream,
+        ServerCallContext context)
+    {
+        switch (request.Type)
+        {
+            case "test":
+                await responseStream.WriteAsync(new FlightResult("test data"));
+                break;
+            case "GetPrimaryKeys":
```

Comment:
We should use the FlightSQL constants for these items `SqlAction.CreateRequest`.  For the ones that don't exist like `BeginTransaction` we should add them.

Reply:
done